frontend/src/NonSports.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+
+class NonSports extends Component {
+  constructor(props){
+    super(props);
+    var isSportsOrAuthor = false;
+
+    //gets the location of the page, if it is a sports or an author page,
+    //the red header will appear
+    if (window.location.href.indexOf("sports") != -1 || window.location.href.indexOf("author") != -1){
```

Comment:
we might want to make this check more robust once we start getting data from the server